shared/src/types/key/mod.rs

```diff
@@ -356,6 +357,26 @@ impl<PK: PublicKey> From<&PK> for PublicKeyHash {
     }
 }
 
+/// Convert validator's consensus key into address raw hash that is compatible
+/// with Tendermint
+pub fn tm_consensus_key_raw_hash(pk: &common::PublicKey) -> String {
+    match pk {
+        common::PublicKey::Ed25519(pk) => {
+            let pkh = PublicKeyHash::from(pk);
+            pkh.0
+        }
+        common::PublicKey::Secp256k1(pk) => {
+            let pkh = PublicKeyHash::from(pk);
+            pkh.0
+        }
+    }
+}
+
+/// Convert Tendermint validator's raw hash bytes to Namada raw hash string
```

Comment:
`validator's raw hash bytes` - not sure what this is, test/example would be useful here

Reply:
Same thing, this got pulled in from main. Can remove if necessary